eva-saga/src/main/kotlin/com/razz/eva/saga/SagaExecutionContext.kt

```diff
@@ -0,0 +1,11 @@
+package com.razz.eva.saga
+
+import io.opentelemetry.api.OpenTelemetry
+
+data class SagaExecutionContext(
+    val otel: OpenTelemetry,
```

Comment:
make field and ctor internal, and let's have `fun sagaExecutionContext(otel: OpenTelemetry) = SagaExecutionContext(otel)`